blogs/lokiless_netobserv/index.md

```diff
@@ -0,0 +1,91 @@
+
+In our perf tests, [kube-burner](https://github.com/kube-burner/kube-burner) workloads were used to generate several objects and create heavy network traffic. We used sampling rate of 1 for all below tests. To further describe each test bed:
+
+1. Test bed 1: node-density-heavy workload ran against 25 nodes cluster.
+2. Test bed 2: ingress-perf workload ran against 65 nodes cluster.
+3. Test bed 3: cluster-densit-v2 workload ran against 120 nodes cluster
+
+Below graphs shows total vCPU and memory usage for a recommended Network Observability stack (flowlogs-pipeline, eBPF-agent, Kafka and optionally Loki) for production clusters.
+
+![Compare total vCPUs utilized with and without Loki](<images/Total vCPUs consumed.png>)
+![Compare total RSS utilized with and without Loki](<images/Total Memory (RSS) consumed.png>)
+
+<sup>*</sup> actual resource utilization may depend on various factors such as flowcollector sampling size, number of workloads and nodes in an OCP cluster
+
+This comes with couple of trade-offs though, without storage of network flows it no longer provides Traffic flows table. Also, per-pod level of resource granularity is not available since it causes prometheus metrics to have high cardinality. However, should you need per-pod or per-flow level of granularity for diagnostic and troubleshooting needs, enabling loki should be pretty straightforward and both datasources can be used. When both loki and prometheus datasources are enabled, while querying on Netflow Traffic page, prometheus queried data over will be used wherever possible since it offers faster performance.
+
+## Accessing Netflow Traffic for non-admins:
+While prometheus currently doesn't supports multi-tenancy in a way that Loki does in an OpenShift cluster, non-admin users can be added to `cluster-monitoring-view`. For example, below command can be used to enable prometheus metrics visualizing for `testuser-0` user.
+
+`oc adm policy add-cluster-role-to-user cluster-monitoring-view  testuser-0`
+
+## Network Observability metrics use case:
+Let's look at a scenario how users can benefit from metrics published by Network Observability Operator. For instance, if you suspect anamaly with DNS lookups in your cluster and want to investigate workloads that may be facing DNS latencies. With Network Observability's `DNSTracking` feature and enriched prometheus metrics you can quickly set up an alert to trigger on high DNS latencies.
+
+For example, below alert will trigger for any workloads that experiences DNS latency > 100ms: 
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: dns-latency-alert
```

Comment:
I don't think it needs to be in particular NS, but I just added this `netobserv` NS since to just organize resource better. I'll check if it's able to trigger when deployed in different NS as well.